docs/modules/ROOT/pages/relay.adoc

```diff
@@ -98,6 +98,16 @@ Every transaction in Defender Relay is valid for submission to the Ethereum netw
 
 `validUntil` defaults to 8 hours after the transaction creation. Note that you can combine validUntil with a fixed gas price to achieve extremely fast mining times and beating other transactions on `gasPrice`.
 
+In order to use `validUntil` with `ethers.js` provide an option `validForSeconds` for `DefenderRelaySigner` there `validForSecods` stands for how many seconds the transaction would be valid after its creation.
+
+[source,jsx]
+----
+const signer = new DefenderRelaySigner(credentials, provider, { validForSeconds: 120 });
+----
+
+
+NOTE: `validUntil` is a UTC timestamp. Make sure to use a UTC timezone not a local one.
```

Comment:
NOTE: `validUntil` is a UTC timestamp. Make sure to use a UTC timezone and not a local one.